frontend/encore/typescript.rst

```diff
@@ -34,6 +34,20 @@ also configure the `ts-loader options`_ via a callback:
 If React assets are enabled (``.enableReactPreset()``), any ``.tsx`` file will be
 processed as well by ``ts-loader``.
 
+
+Here is a sample ``tsconfig.json`` file :
+
+.. code-block:: javascript 
+
+{
```

Comment:
all block should be indented, isn't it?
what about Encore v0.21: is this still valid now?

Reply:
So gave this a try on the 0.21.1 version of webpack-encore. It still complains with following message :

```
TS18002: The 'files' list in config file 'tsconfig.json' is empty.
```

Creating this file with this default content solved the problem. Plus most people, will customize their Typescript experience in the `tsconfig` file. Perhaps, this should even be fixed as a hook in the recipe of the new bundle.